kinetic/distribution.yaml

```diff
@@ -13875,6 +13875,16 @@ repositories:
       url: https://github.com/ethz-asl/ethzasl_xsens_driver.git
       version: master
     status: maintained
+  xsens_official:
+    doc:
+      type: git
+      url: https://github.com/xsens/xsens_mti_ros_node.git
```

Comment:
Related i'll note that this repository name is a little misleading as it's more than just the ROS node.

Reply:
Thank you for the feedback. Should it be named driver instead of node? Or do you suggest something else?

Reply:
Often the repository will match the package name, but that's generally only done if there's only one package in the repository. I'd suggest `xsens_support` or something else similar. Or `xsens_ros_driver` maybe.